test/ClangModules/CoreGraphics_test.swift

```diff
@@ -1,24 +1,27 @@
-// RUN: %target-swift-frontend -emit-ir -O %s | FileCheck %s
+// RUN: %target-swift-frontend -target x86_64-apple-macosx10.11 -emit-ir -O %s | FileCheck %s
```

Comment:
@milseman Why is an explicit target needed?  Could you use an availability check in the test instead?